.github/aznfs-build.yaml

```diff
@@ -55,6 +55,7 @@ stages:
               export STG_DIR=$(System.DefaultWorkingDirectory)
               export SOURCE_DIR=$(System.DefaultWorkingDirectory)
               export BUILD_TYPE=${{ parameters.buildType }}
+              export BUILD_MACHINE=ubuntu
```

Comment:
lets add one more parameters.DynamicLinking type boolean and read here that. IN that way we can build Dynamic linking for ubuntu as well as Azure linux.